src/core/dbus.c

```diff
@@ -977,7 +977,7 @@ static int bus_init_private(Manager *m) {
                 char *p = sa.un.sun_path;
                 const char *e;
 
-                e = secure_getenv("XDG_RUNTIME_DIR");
+                e = getenv("XDG_RUNTIME_DIR");
```

Comment:
well, what I am saying is that systemd makes use of sd-bus, and hence secure_getenv() at numerous places, and that's not going to change. Changing one occurence of it is not going to cut it, as the library code the same process uses uses the very same expression secure_getenv("XDG_RUNTIME_DIR") all over the place.